ui/imports/shared/popups/walletconnect/DAppsListPopup.qml

```diff
@@ -88,7 +88,15 @@ Popup {
             id: listViewWrapper
             Layout.fillWidth: true
             Layout.preferredHeight: listView.contentHeight
-            Layout.maximumHeight: 290
+
+            // TODO: uncomment maximumHeight: 290 and remove fillHeight: true
```

Comment:
This will be probably forgotten, could you please mention the issue (#<number>) that tracks the status of the upgrade so that you can search by issue ID and find the place that requires changes?

Reply:
Mentioned #15595 as the issue where the problem occurs. Unfortunately, it looks like on the latest Qt5.15.13 on mac this problem still is easily reproducible.